cdap-security/pom.xml

```diff
@@ -160,8 +160,17 @@
       <scope>test</scope>
       <type>test-jar</type>
     </dependency>
+    <dependency>
+      <groupId>com.google.crypto.tink</groupId>
+      <artifactId>tink</artifactId>
+      <version>1.6.0</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
```

Comment:
Assuming this dep is required for `tink` crypto lib?

Reply:
Removed. No needed to be explicitly added.